foundry.toml

```diff
@@ -82,7 +82,7 @@ localanvil = "${ETH_NODE_URI_LOCALANVIL}"
 [etherscan]
 mainnet = { key = "${MAINNET_ETHERSCAN_API_KEY}", url = "https://api.etherscan.io/v2/api?chainid=1", chain = "1" }
 abstract = { key = "${ABSTRACT_ETHERSCAN_API_KEY}", url = "https://api.abscan.org/api", chain = "2741" }
-apechain = { key = "${APECHAIN_ETHERSCAN_API_KEY}", url = "https://api.apescan.io/api", chain = "33139" }
+apechain = { key = "${APECHAIN_ETHERSCAN_API_KEY}", url = "https://api.etherscan.io/v2/api?chainid=33139", chain = "33139" }
```

Comment:
if we want to verify via mainnet etherscan then we should also use the MAINNET_ETHERSCAN_API_KEY

Reply:
true!